package-lock.json

```diff
@@ -1,8 +1,19342 @@
 {
   "name": "paraview-medical",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Here's the culprit - I triggered a new lockfile version by using npm 7.6.0 here. It may be now problem, it's backwards compatible: https://stackoverflow.com/questions/66150813/is-lockfileversion-2-in-package-lock-json-from-npm-7-compatible-with-older-ve

Reply:
Alternatively I may downgrade npm and reinstall.